test/CommandLineUtils.Tests/CommandLineApplicationOfTTests.cs

```diff
@@ -111,6 +111,7 @@ public void ItDoesNotInitalizeClassUnlessNecessary()
             Assert.Same(app, parseResult.SelectedCommand);
         }
 
+        [Command]
```

Comment:
Can you undo all the places this had to be added? I’d like to ensure we maintain backwards compatibility as much as possible.

Reply:
Removed where added.